docs/reference/glossary.md

```diff
@@ -32,6 +32,15 @@ Their `slug` property begins with `/`, which is only true for pages. A page also
 
 `path` differs from `slug` in that it always reflects the true parent-child relationships between pages in the tree, while `slug` can be edited and shortened if desired so that URLs don't have to contain a lot of slashes to reach a deep page.
 
+## Index page
+
+An index page, is a special [page](#page) that is built to display a listing of [pieces](#piece). Index pages extend the `@apostrophecms/piece-page-type` module. In addition to data available to all pages, index page templates have access to a `data.pieces` array, which contains objects representing individual pieces, limited by the pieces page `perPage` option and the state of pagination. Index pages also have `data.currentPage` and `data.totalPages` number values available to support pagination.
+
+## Show page
+
+Despite their name these are *not* actually [pages](#page) in the technical sense. They are a feature, enabled by creating an [index page](#index-page), providing individual URLs that display a rendered template for individual [pieces](#piece). **A blog listing is an "index page" and the individual blog post is a "show page."**
```

Comment:
How about:

"Despite their name these are not represented as separate pages in Apostrophe's page tree."

Reply:
I'm more concerned with pointing out that they don't extend the page module.